docs/docs/api/Tasks/getTasks.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Get All Tasks
+---
+
+export const Endpoint = ({children, color}) => ( <span style={{
+borderRadius: '2px',
+color: '#E83E8C',
+}}>{children}</span> );
+
+<Endpoint>GET tasks/</Endpoint>: Get all tasks <br></br>
```

Comment:
Minor comment: Similar to the other comments on the other pages, please add the `/` on your endpoints (for consistency)